app/src/main/java/cs465/illinois/edu/dogonthequad/CreateMeetupActivity.java

```diff
@@ -0,0 +1,69 @@
+package cs465.illinois.edu.dogonthequad;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+
+import com.google.gson.Gson;
+
+import java.lang.reflect.Type;
+
+import static cs465.illinois.edu.dogonthequad.MapActivity.MEETUP_KEY;
+
+public class CreateMeetupActivity extends Activity implements View.OnClickListener{
+
+    public Meetup mMeetup;
+    public Class mNextActivity;
```

Comment:
Why not make a function `getNextActivity()` that figures out what the next one will be? Then each subclass isn't responsible for knowing which activity is next.

Reply:
Because it's kind of non-trivial either way, if you want to make the change feel free.